### PR TITLE
Add TracerBuilderExtensions to configure LightStep exporter

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
Adds `UseLightStep` extension method to TracerBuilder.

### CODE REVIEW

**Comment by MikeGoldsmith** [comment]
Seems my VSCode wants to change the whole README - I'll try to fix that.

File: README.md [hunk 1]
```diff
@@ -1,672 +1,672 @@
...
+using (var tracerFactory = TracerFactory.Create(
+    builder => builder.UseLightStep(o =>
+        {
+            o.ServiceName = "my-service"
+            o.AccessToken = "access-token";
+        })))
+{
+    var tracer = tracerFactory.GetTracer("lightstep-test");
+    var span = tracer
+        .SpanBuilder("incoming request")
```
Comment (lmolkova) [inline README.md]: I guess this is out of date, we don't have SpanBuilder anymore.

**Comment by lmolkova** [comment]
Here is the link to [LightStep sample](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/samples/Exporters/TestLightstep.cs) that is nice to update too

**Comment by MikeGoldsmith** [comment]
Opps - I must have copied the wrong local example. I'll fix the example in the README.

**Pull request merged**